### PR TITLE
Expose get_segment_used_size

### PR DESCRIPTION
**Pull request by mmsc2**

### CODE REVIEW

File: src/vm/vm_core.rs [hunk 1]
```diff
@@ -3302,4 +3306,26 @@ mod tests {
...
+            ((0, 2), 1),
+            ((0, 5), 1),
+            ((0, 7), 1),
+            ((1, 1), 1),
+            ((2, 2), 1),
+            ((2, 4), 1),
+            ((2, 7), 1)
+        ];
+        vm.segments.compute_effective_sizes(&vm.memory);
+        assert_eq!(Some(8), vm.segments.get_segment_used_size(2));
```
Comment (fmoletta) [inline src/vm/vm_core.rs:3323]:
```suggestion
        assert_eq!(Some(8), vm.get_segment_used_size(2));
```
The new method is not being tested

File: src/vm/vm_core.rs [hunk 2]
```diff
@@ -3302,4 +3306,26 @@ mod tests {
...
+    }
+
+    #[test]
+    fn get_segment_used_size_before_computing_used() {
+        let vm = vm!();
+        assert_eq!(None, vm.segments.get_segment_used_size(2));
```
Comment (fmoletta) [inline src/vm/vm_core.rs:3329]:
```suggestion
        assert_eq!(None, vm.get_segment_used_size(2));
```
Same here

**Review by fmoletta — CHANGES_REQUESTED** [review]

File: src/vm/vm_core.rs (hunk 1 above)
Comment (mmsc2) [reply src/vm/vm_core.rs]: Fixed

**Review by juanbono — APPROVED** [review]

**Review by fmoletta — APPROVED** [review]
lgtm

**Pull request merged**